ledfx/events.py

```diff
@@ -224,19 +227,35 @@ def __init__(
 class EffectSetEvent(Event):
     """Event emitted when an effect is set or updated"""
 
-    def __init__(self, effect_name, effect_id, effect_config, virtual_id):
+    def __init__(
+        self,
+        effect_name: str,
+        effect_id: str,
+        effect_config: dict[str, Any],
```

Comment:
I worry that sending the full config can be a bit spammy.

If a client cares, it can ask for the config of the effect.

Reply:
Ok, that's fine with me.

Two integrations were filtering on event_name but I've migrated them to filter on event_id instead.